CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewTopComponent.java

```diff
@@ -53,6 +53,8 @@
     "HINT_QualityControlViewTopComponent=Quality Control View"})
 public final class QualityControlViewTopComponent extends JavaFxTopComponent<QualityControlViewPane> implements QualityControlListener {
 
+    private static boolean firstClick = true;
```

Comment:
This variable should be defined in the QualityControlViewPane class.
There's no need for any code changes in this class.